core/iwasm/compilation/aot_emit_aot_file.c

```diff
@@ -4343,6 +4335,12 @@ aot_obj_data_create(AOTCompContext *comp_ctx)
         /* close and remove temp file */
         close(fd);
         unlink(file_name);
+#else
+        if (_mktemp_s(file_name, sizeof(file_name))) {
+            aot_set_last_error("make temp file failed.");
+            goto fail;
+        }
+#endif
```

Comment:
Please add a `os_mktemp()` in *core/shared/platform/include/platform_api_extension.h* and platform-specific implantation in *core/shared/platform*.

Reply:
It seems a little strange to add this to `core/shared/platform/include/platform_api_extension.h`, it says "the Fileystem APIs are required for WASI libc support", but mktemp is only used in aot compiler rather than runtime.